### PR TITLE
Fix saving autosave to api (PS-951)

### PR DESCRIPTION
**Pull request by JakubMatejka**

### CODE REVIEW

File: notebookUtils.py [hunk 1]
```diff
@@ -60,15 +60,21 @@ def saveFile(file_path, token, log):
...
 
-    headers = {'X-StorageApi-Token': token, 'User-Agent': 'Keboola Sandbox Autosave Request'}
+    headers = {
+        'Content-Type': 'application/json',
+        'User-Agent': 'Keboola Sandbox Autosave Request',
+        'X-StorageApi-Token': token,
+    }
     url = os.environ['SANDBOXES_API_URL'] + '/sandboxes/' + os.environ['SANDBOX_ID']
-    json = {'lastAutosaveTimestamp': datetime.now().strftime('%Y-%m-%dT%H:%M:%S.%f')}
-    retrySession().put(url, json, headers=headers)
```
Comment (JakubMatejka) [inline notebookUtils.py:71]: This `json` parameter was the problem, the requests lib did not translate it correctly. I also added logging of API errors.

**Review by pivnicek — APPROVED** [review]
LGTM

**Comment by JakubMatejka** [comment]
@pivnicek hmm, it seems to be installed from git directly so I just force start its release pipeline, right? https://github.com/keboola/sandboxes-jupyterlab/blob/master/common/Dockerfile#L135

**Comment by pivnicek** [comment]
@JakubMatejka yes, it pulls `latest` (maybe we should change that? :-) .  So it needs a rebuild but no source change.

**Comment by JakubMatejka** [comment]
Right, I'll rather fix the version in the Dockerfile.

**Pull request merged**